dev/tests/integration/testsuite/Magento/ProductAlert/_files/customer_unsubscribe_stock.php

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
You should also write `dev/tests/integration/testsuite/Magento/ProductAlert/_files/customer_unsubscribe_stock_rollback.php` file cleaning the inserted entries.

Reply:
Hello, thanks for the suggestion.
The file was added.